linux_os/guide/system/apparmor/apparmor_configured/rule.yml

```diff
@@ -0,0 +1,74 @@
+documentation_complete: true
+
+prodtype: sle12,sle15
+
+title: 'Ensure AppArmor is Active and Configured'
+
+description: |-
+    Verify that the SUSE operating system Apparmor tool is configured to
+    control whitelisted applications and user home directory access
+    control.<br/><br/>
+
+    Check that <tt>pam_apparmor</tt> is installed on the system with the following
+    command:
+
+    <pre># rpm -q pam_apparmor</pre>
+
+    Check that the "apparmor" daemon is running with the following command:
+
+    <pre># systemctl status apparmor.service | grep -i active
+
+    Active: active (exited) since Fri 2017-01-13 01:01:01 GMT; 1day 1h ago</pre>
+
+    <strong>Note:</strong> <tt>pam_apparmor</tt> must have properly configured
+    profiles. All configurations will be based on the actual system setup and
+    organization.  See the <tt>pam_apparmor</tt> documentation for more
+    information on configuring profiles.
+
+rationale: |-
+    Using a whitelist provides a configuration management method for allowing
+    the execution of only authorized software. Using only authorized software
+    decreases risk by limiting the number of potential vulnerabilities.<br/><br/>
+
+    The organization must identify authorized software programs and permit
+    execution of authorized software by adding each authorized program to the
+    "pam_apparmor" exception policy. The process used to identify software
+    programs that are authorized to execute on organizational information
+    systems is commonly referred to as whitelisting.<br/><br/>
+
+    Verification of whitelisted software occurs prior to execution or at system
+    startup.<br/><br/>
+
+    Users' home directories/folders may contain information of a sensitive
+    nature. Nonprivileged users should coordinate any sharing of information
+    with a System Administrator (SA) through shared resources.<br/><br/>
+
+    Apparmor can confine users to their home directory, not allowing them to
+    make any changes outside of their own home directories. Confining users to
+    their home directory will minimize the risk of sharing information.
+
+severity: medium
+
+identifiers:
+    cce@sle12: CCE-83194-1
+    cce@sle15: CCE-85752-4
+
+references:
+    disa: CCI-001764,CCI-001774,CCI-002165,CCI-002233,CCI-002235
+    nist: AC-3(4),AC-6(8),AC-6(10),CM-7(5)(b),CM-7(2),SC-7(21),CM-6(a)
+    srg: SRG-OS-000312-GPOS-00122,SRG-OS-000312-GPOS-00123SRG-OS-000312-GPOS-00124,SRG-OS-000324-GPOS-00125,SRG-OS-000326-GPOS-00126,SRG-OS-000370-GPOS-00155,SRG-OS-000480-GPOS-00230,SRG-OS-000480-GPOS-00227,SRG-OS-000480-GPOS-00231,SRG-OS-000480-GPOS-00232
+    stigid@sle12: SLES-12-010600
+    stigid@sle15: SLES-15-010390
+
+ocil_clause: 'it is not'
+
+
+ocil: '{{{ describe_service_enable(service="apparmor") }}}'
```

Comment:
Unfortunately for this macro you should use the multiline approach, otherwise the build fails when building rhcos4 content even though this content is not enabled for it (macros are still rendered before checking if `prodtype` contains certain product or not).
```suggestion
ocil: |-
    {{{ describe_service_enable(service="apparmor") }}}
```

Also, maybe a good idea here would be to use 
```
ocil: |-
    {{{ ocil_service_enabled(service="apparmor") }}}
```
which is a better fit for OCIL content.